airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/commands.py

```diff
@@ -166,6 +166,7 @@ async def test(
             run_step_options=run_step_options,
             targeted_platforms=[LOCAL_BUILD_PLATFORM],
             secret_stores=ctx.obj["secret_stores"],
+            enable_report_auto_open=ctx.obj.get("enable_report_auto_open", True),
```

Comment:
Feels like this default should be false, I am sensing this generally generates more frustration for others around me as well.